src/Material.cpp

```diff
@@ -1039,6 +1039,37 @@ Material* Material::clone() {
 
 
 /**
+ * @brief Copy another Material's cross section data into this Material.
+ * @details This method performs a deep copy of the other Material's
+ *          cross section data arrays for use by this Material.
+ * @param material a pointer to the Material with data to copy
+ */
+void Material::copy(Material* material) {
```

Comment:
Since this method doesn't return a copy (what one might expect from a copy method) but instead copies *from* the input material to the current material, I would recommend naming this method `copyFrom(...)`?